src/services/user/user.class.js

```diff
@@ -43,11 +43,44 @@ class Service {
       };
     });
 
+    let level = 1;
```

Comment:
Is there a use case where you would want to start at level 0?

Reply:
I also thought about that, but couldn't think of a use case. I haven't played a single video game where you start at level 0, I think.